pkg/cmd/server/origin/asset_apiserver_adapter.go

```diff
@@ -0,0 +1,68 @@
+package origin
+
+import (
+	"os"
+
+	assetapiserver "github.com/openshift/origin/pkg/assets/apiserver"
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	"github.com/openshift/origin/pkg/cmd/util/pluginconfig"
+	"github.com/openshift/origin/pkg/quota/admission/clusterresourceoverride"
+	clusterresourceoverrideapi "github.com/openshift/origin/pkg/quota/admission/clusterresourceoverride/api"
+)
+
+// TODO this is taking a very large config for a small piece of it.  The information must be broken up at some point so that
+// we can run this in a pod.  This is an indication of leaky abstraction because it spent too much time in openshift start
+func NewAssetServerConfigFromMasterConfig(masterConfigOptions configapi.MasterConfig) (*assetapiserver.AssetServerConfig, error) {
```

Comment:
Everything here is a target for removal.  Looks like we have some cross-over cruft with part of the admission chain.